tests/integration/pilot/main_test.go

```diff
@@ -41,7 +41,20 @@ var (
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
-		Setup(istio.Setup(&i, nil)).
+		Setup(istio.Setup(&i, func(c resource.Context, cfg *istio.Config) {
+			if c.Settings().EnableDualStack {
```

Comment:
This should probably be handled at the framework level instead of each  test. but lgtm